server/game/cards/07_LAW/units/JabbasRancorSnackTime.ts

```diff
@@ -0,0 +1,33 @@
+import type { IAbilityHelper } from '../../../AbilityHelper';
+import type { INonLeaderUnitAbilityRegistrar } from '../../../core/card/AbilityRegistrationInterfaces';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, WildcardCardType, ZoneName } from '../../../core/Constants';
+
+export default class JabbasRancorSnackTime extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: 'jabbas-rancor#snack-time-id',
+            internalName: 'jabbas-rancor#snack-time',
+        };
+    }
+
+    public override setupCardAbilities(registrar: INonLeaderUnitAbilityRegistrar, abilityHelper: IAbilityHelper) {
+        registrar.addOnAttackAbility({
+            title: 'An opponent chooses a ground unit they control. You may deal 7 damage to that unit',
+            targetResolver: {
+                controller: RelativePlayer.Opponent,
+                choosingPlayer: RelativePlayer.Opponent,
+                cardTypeFilter: WildcardCardType.Unit,
+                zoneFilter: ZoneName.GroundArena,
+            },
```

Comment:
Since the opponent is the one being prompted, we can add an `activePromptTitle:` to the target resolver that reads better from their perspective.